### DISCUSSION TITLE
Can't install LHC in my website.

### DISCUSSION THREAD

**Discussion by petraamaral** [LiveHelperChat/livehelperchat#2037] [Q&A]
Hello,
I am trying to install LHC in my website, it's hosted in a service provider here in BR. I have PHP 8.0, MySQL 5.7. My devs can't find the solution so I am at a lost. I'll put the log down below, if any other information is needed please ask away.
[default.log](https://github.com/LiveHelperChat/livehelperchat/files/13922148/default.log)

**Comment by remdex** [comment]
This error tells all...

Call to undefined function php_strip_whitespace

**Reply by petraamaral** [reply]
Ok, but I'm not a DEV and the ones I have working rn are not PHP dvs. Is there an easy fix? What can I do?

**Reply by remdex** [reply]
Error tells it all. Forward it your hosting company and ask why this function is not found...
